### PR TITLE
Automated acceptance tests

### PR DESCRIPTION
**Pull request by iximiuz**
[Fixes-###](https://github.com/docker-slim/docker-slim/issues/###)
==================================================================

What
===============

Why
===============

How Tested
===============

### CODE REVIEW

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+name: build-and-test
+on:
+  push:
+    branches:
+      - master
+      - iximiuz/github-actions
```
Comment (D4N) [inline .github/workflows/build-and-test.yml:6]: This is probably only included for testing, right?
```suggestion
```

File: .github/workflows/build-and-test.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out docker-slim/docker-slim repo
+        uses: actions/checkout@v3
+
+      - name: Set up Go development environment
+        uses: actions/setup-go@v3
+        with:
+          go-version: '1.15.0'
```
Comment (D4N) [inline .github/workflows/build-and-test.yml:19]: Would it make sense to try and build docker slim for all supported go versions?

File: .github/workflows/build-and-test.yml (hunk 1 above)
Comment (iximiuz) [reply .github/workflows/build-and-test.yml:6]: Hi, Dan! 😄  Yes, this MR is still WiP and it's supposed to work in pair with [this docker-slim/examples MR](https://github.com/docker-slim/examples/pull/22).

File: .github/workflows/build-and-test.yml (hunk 2 above)
Comment (iximiuz) [reply .github/workflows/build-and-test.yml:19]: To be honest, I don't know. Until this MR, `go.mod` had `go 1.13`, but I couldn't actually build the project with anything older than `1.15`. So, I bumped the version up. However, I'm not aware of the inverse scenarios when something wouldn't build with a newer Go version than specified in the `go.mod`. The language is kind of famous for its backward compatibility guarantees. But I'm not 100% sure 😄

Comment (iximiuz) [reply .github/workflows/build-and-test.yml:19]: Good point about the `latest`!

**Pull request merged**